sqlite_utils/cli.py

```diff
@@ -1374,6 +1378,15 @@ def create_database(path, enable_wal):
     db = sqlite_utils.Database(path)
     if enable_wal:
         db.enable_wal()
+
+    # load spatialite from expected locations
+    if init_spatialite:
+        db.init_spatialite()
+
+    # load spatialite or another extension from a custom location
+    if load_extension:
+        _load_extensions(db, load_extension)
```

Comment:
I think this needs to happen before the call to `db.init_spatialite()` to ensure SpatiaLite is correctly loaded in the case where it wasn't in an expected location so the user had to use `--load-extension` with it.

Reply:
Switched. Good catch.